packages/teams-js/src/internal/validDomains.ts

```diff
@@ -0,0 +1,104 @@
+import { GlobalVars } from '../internal/globalVars';
+import { validDomainsCdnEndpoint, validOriginsFallback } from './constants';
+import { getLogger } from './telemetry';
+import { isValidHttpsURL } from './utils';
+
+let validOrigins: string[] = [];
+export async function prefetchDomainsFromCDN(): Promise<void> {
+  if (fetch) {
```

Comment:
>  if (fetch) [](http://example.com/codeflow?start=0&length=12)

Can you say more why you have to check for the presence of the `fetch` function? Surely any modern browser we support has implemented this.

Reply:
This is for SSR/windowless scenarios